prac_06/guitar_test.py

```diff
@@ -0,0 +1,19 @@
+"""
+CP1404/CP5632 Practical
+Guitar Test
+
+Expected Time: 10 Minutes
+Actual Time: 4 Minutes
+"""
+from prac_06.guitar import Guitar
+
+gibson = Guitar("Gibson L-5 CES", 1922, 16035.40)
+stratocaster = Guitar("Fender Stratocaster American Pro", 2021, 2510.28)
+
+print(gibson)
+print(f"Age: {gibson.get_age()}")  # Expected 101, got 101
```

Comment:
if the gibson.get_age() returns the wrong value, could we detect from the printout?

Reply:
You could if you knew what the expected value was.